Failing test: `TestSignTransaction`

After a rebase (celo5) the new test `TestSignTransaction` fails with:
```
--- FAIL: TestSignTransaction (0.00s)
   ./op-geth/internal/ethapi/api_test.go:1176: result mismatch. Have:
        {"type":"0x2","chainId":"0x1","nonce":"0x0","to":"0x703c4b2bd70c169f5717101caee543299fc946c7","gas":"0x5208","gasPrice":null,"maxPriorityFeePerGas":"0x0","maxFeePerGas":"0x684ee180","value":"0x1","input":"0x","accessList":[],"v":"0x0","r":"0x8fabeb142d585dd9247f459f7e6fe77e2520c88d50ba5d220da1533cea8b34e1","s":"0x582dd68b21aef36ba23f34e49607329c20d981d30404daf749077f5606785ce7","yParity":"0x0","hash":"0x93927839207cfbec395da84b8a2bc38b7b65d2cb2819e9fef1f091f5b1d4cc8f","feeCurrency":null}
        Want:
        {"type":"0x2","chainId":"0x1","nonce":"0x0","to":"0x703c4b2bd70c169f5717101caee543299fc946c7","gas":"0x5208","gasPrice":null,"maxPriorityFeePerGas":"0x0","maxFeePerGas":"0x684ee180","value":"0x1","input":"0x","accessList":[],"v":"0x0","r":"0x8fabeb142d585dd9247f459f7e6fe77e2520c88d50ba5d220da1533cea8b34e1","s":"0x582dd68b21aef36ba23f34e49607329c20d981d30404daf749077f5606785ce7","yParity":"0x0","hash":"0x93927839207cfbec395da84b8a2bc38b7b65d2cb2819e9fef1f091f5b1d4cc8f"}
```
        
The problem is that `"feeCurrency":null` gets added to the json-ified version of the transaction, even though it shouldn't.

See also #109

--- a/core/types/transaction_marshalling.go
+++ b/core/types/transaction_marshalling.go
@@ -65,7 +65,7 @@ type txJSON struct {
 	Hash common.Hash `json:"hash"`
 
 	// Celo specific fields
-	FeeCurrency *common.Address `json:"feeCurrency"` // nil means native currency
+	FeeCurrency *common.Address `json:"feeCurrency,omitempty"` // nil means native currency
 }
 
 // yParityValue returns the YParity value from JSON. For backwards-compatibility reasons,

--- a/internal/ethapi/api_test.go
+++ b/internal/ethapi/api_test.go
@@ -1142,7 +1142,6 @@ func TestCall(t *testing.T) {
 }
 
 func TestSignTransaction(t *testing.T) {
-	t.Skip("Currently failing, see https://github.com/celo-org/op-geth/issues/117")
 	t.Parallel()
 	// Initialize test accounts
 	var (

